app/cafe.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+from app.errors import *
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            vaccine_info = visitor["vaccine"]
+        except KeyError:
+            raise NotVaccinatedError
+        try:
+            expiration_date = vaccine_info["expiration_date"]
+            if expiration_date < datetime.date.today():
+                raise OutdatedVaccineError
+        except KeyError:
+            raise NotVaccinatedError
```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. Consider using: `raise NotVaccinatedError('Visitor is not vaccinated')`.